src/main/java/org/mockito/internal/matchers/apachecommons/EqualsBuilder.java

```diff
@@ -14,6 +11,7 @@
 import java.util.Collections;
 import java.util.List;
 
+// Class comes from Apache Commons Lang, added some tiny changes
```

Comment:
I wonder if it is ok to re-license the class on the Apache license to MIT.

Reply:
I am not sure. We should probably get rid of these classes and https://static.javadoc.io/org.mockito/mockito-core/3.1.0/org/mockito/ArgumentMatchers.html#refEq-T-java.lang.String...- as a whole. Filed #1800 to track that effort.